src/cml.js

```diff
@@ -464,7 +465,7 @@ class CML {
       await exec(`git checkout -b ${source}`);
       await exec(`git add ${paths.join(' ')}`);
       let commitMessage = `CML PR for ${shaShort}`;
-      if (!(merge || rebase || squash)) {
+      if ( skipCI ){
```

Comment:
We want cml pr to still do skipCI if not automerge

```suggestion
      if ( skipCI ||  !(merge || rebase || squash)){
```